data/airlines/data.json

```
@@ -0,0 +1,214 @@
+[
+  { "name": "Aegean Airlines", "iata": "A3" },
+  { "name": "Aer Lingus Group Plc.", "iata": "El" },
```

IATA should be EL https://www.iata.org/en/publications/directories/code-search/?airline.search=Aer%20Lingus

---

I think it's already correct 
```suggestion
  { "name": "Aer Lingus Group Plc.", "iata": "EI" },
```